src/App.js

```diff
@@ -46,14 +46,8 @@ function App() {
       </header>
 
       <body className='body'>
-          <div className='image_body1'>
-            <img src='./image/Hapo_Learn_banner 1.png'></img>
-          </div>
-
-          <div className='p1_1'>
-              <p>Learn Anytime, Anywhere </p>
-          </div>
-
+          <div className='image_body1'><img src='./image/Hapo_Learn_banner 1.png'></img></div>
```

Comment:
Sửa lại hết tên của class cho đúng chuẩn format của wiki.

Reply:
Bên dưới cũng tương tự. Check lại hết

Reply:
vâng để xem chỉnh ạ

Reply:
em chỉnh được 1 tí lú luôn anh toàn code là code huhu